content/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "Addressing hybrid cloud application challenges using HPE GreenLake for
+  Private Cloud Enterprise – Part 1: Deploying complex apps"
+date: 2023-01-02T07:52:05.869Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - hpe-greenlake, hpe-greenlake-for-private-cloud-enterprise, Skupper, AWS EKS
+---
+## Introduction
+
+[HPE GreenLake for Private Cloud Enterprise](https://www.hpe.com/us/en/greenlake/private-cloud-enterprise.html) delivers a modern private cloud to support your app workloads with bare metal, containers, and virtual machines (VMs) running in any combination across your edges, colocations, and data centers. It combines self-service resource access for developers with consumption and performance transparency for IT. 
+
+This blog post show you how to deploy a complex application that consists of multiple microservices as a hybrid app that spans both public AWS EKS cluster and private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise. By using a hybrid cloud solution, you can combine the compliance benefits of a private cloud in HPE GreenLake for Private Cloud Enterprise environment with the scalability and connectivity of the public cloud. You can rely on the security of finely tuned, on-premises data centers while turning to the agility of cloud computing to manage the front end of an application in the public cloud. It essentially can optimize resource allocation, save costs and improve overall productivity and performance in the process. 
+
+## Prerequisites
+
+Before you start, make sure you meet the following requirements: 
+
+* A public Kubernetes cluster from one of the public cloud providers such as *AWS*, *Microsoft Azure* or *Google*. We use one EKS cluster, named *eks-cfe-public* from AWS, in this blog for application deployment. However, it works if you choose a cluster from other providers.
```

Comment:
I'd recommend rephrasing the second sentence here to: "For the purposes of the use case being highlighted in this blog post, a single EKS cluster, named *eks-cfe-public* from AWS, is being used."